src/app/base/components/heuristic-statuses/heuristic-statuses.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'kern-heuristic-statuses',
+  templateUrl: './heuristic-statuses.component.html',
+  styleUrls: ['./heuristic-statuses.component.scss']
+})
+export class HeuristicStatusesComponent implements OnInit {
```

Comment:
if its not used why is OnInit implemented?

Reply:
Removed